vector/src/main/java/im/vector/adapters/VectorMessagesAdapterMediasHelper.java

```diff
@@ -272,19 +273,23 @@ void managePendingImageVideoDownload(final View convertView, final Event event,
         if (null == downloadId) {
             if (message instanceof VideoMessage) {
                 downloadId = mMediasCache.downloadIdFromUrl(((VideoMessage) message).getUrl());
-            } else if (message instanceof ImageMessage) {
-                if (event.getType().equals(Event.EVENT_TYPE_STICKER)) {
-                    downloadId = mMediasCache.downloadIdStickerFromUrl(((StickerMessage) message).getUrl());
-                } else {
-                    downloadId = mMediasCache.downloadIdFromUrl(((ImageMessage) message).getUrl());
-                }
+            } else if (!event.getType().equals(Event.EVENT_TYPE_STICKER) && (message instanceof ImageMessage)) {
+                downloadId = mMediasCache.downloadIdFromUrl(((ImageMessage) message).getUrl());
+            } else if (event.getType().equals(Event.EVENT_TYPE_STICKER) && (message instanceof StickerMessage)) {
```

Comment:
Remove test on event type and test if message is a StickerMessage before ImageMessage